soco/events.py

```diff
@@ -299,87 +142,68 @@ def run(self):
             listener.handle_request()
```

Comment:
I know you didn't touch this code, but could you please rename `listener` to `server` or someting, since it's an `EventServer` and not an `EventListener`.